.github/workflows/cypress-test-multiauth-e2e.yml

```diff
@@ -82,14 +83,30 @@ jobs:
           opensearch_security.multitenancy.tenants.preferred: ["Private", "Global"]
           opensearch_security.readonly_mode.roles: ["kibana_read_only"]
           opensearch_security.cookie.secure: false
-          server.xsrf.allowlist: ["/_plugins/_security/api/authtoken", "/_opendistro/_security/api/authtoken", "/_opendistro/_security/saml/acs", "/_opendistro/_security/saml/acs/idpinitiated", "/_opendistro/_security/saml/logout"]
```

Comment:
Maybe I'm missing context, but why the two sets of /_plugins vs. /_opendistro endpoints? I have seen this a few places and wondering if you can point me to documentation/ do we have any intention of deprecating opendistro endpoints in the future?

Reply:
More out of curiosity, I think if the tests pass it's fine

Reply:
we support Legacy i.e. `/_opendistro/` and current i.e. `/_plugins/` endpoints. Hence xsrf should allow both or none.